dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExport.java

```diff
@@ -260,51 +260,30 @@ public Integer export() throws Exception {
 
         if (container.getImagePullPolicy() != null) {
             var imagePullPolicy = container.getImagePullPolicy().getValue();
-            if (runtime == RuntimeType.quarkus) {
-                imagePullPolicy = StringHelper.camelCaseToDash(imagePullPolicy);
-            }
             buildProperties.add("%s.kubernetes.image-pull-policy=%s".formatted(propPrefix, imagePullPolicy));
         }
 
         // Quarkus Runtime specific
         if (runtime == RuntimeType.quarkus) {
 
             // Quarkus specific dependencies
+            // [TODO] migrate to JKube
```

Comment:
Is this still a TODO?

Reply:
It is part of https://issues.apache.org/jira/browse/CAMEL-21013 which I'm also working on

Reply:
should we the wait for CAMEL-21013 to be resolved first? Otherwise this PR might break the deployment to OpenShift right?

Reply:
Sure, we could wait until openshift is done as well. If we're not going to release any time soon, I'd prefer to do this iteratively and leave openshift out for now. I can continue to look at it next week - today I couldn't get cluster through the chat bot.